charts/fluent-operator/values.yaml

```diff
@@ -171,6 +171,7 @@ fluentbit:
       enable: true
       refreshIntervalSeconds: 10
       memBufLimit: 100MB
+      bufferMaxSize: ""
```

Comment:
Wouldn't it be better to give a default value and then comment it out to keep things uniform?

Reply:
This field is [omitempty](https://github.com/fluent/fluent-operator/blob/master/apis/fluentbit/v1alpha2/plugins/input/tail_types.go#L27C2-L27C15), and the default value is [empty](https://github.com/fluent/fluent-operator/blob/master/apis/fluentbit/v1alpha2/plugins/input/tail_types.go#L113) which means it's ok to set it to ""